database_size/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright 2025 Opener B.V. <https://opener.amsterdam>
```

Comment:
There should be no copyright line in init files.

Reply:
Fixed, thanks!